### ISSUE TITLE
Searching for lib files in root directory

### ISSUE THREAD

**Issue by Bibliofile** [TypeStrong/typedoc#464]
I am running the latest (0.5.9) version of typedoc. Node version 7.8.0

Output: 
```
> typedoc --out docs --mode modules src

Using TypeScript 2.2.1 from C:\...\node_modules\typescript\lib
Error: File 'C:/lib.dom.d.ts' not found.
Error: File 'C:/lib.es2016.d.ts' not found.
```

My `tsconfig.json` file is the following: 
```json
{
    "compilerOptions": {
        "target": "es6",
        "moduleResolution": "node",
        "module": "commonjs",
        "lib": ["es2016", "dom"],
        "noImplicitAny": true,
        "removeComments": false,
        "strictNullChecks": true
    }
}
```

Please let me know if any further information is needed, for now I've just swapped `arr.includes` for `!!~arr.indexOf`, though I'd much rather use the readable alternative.

**Comment by aciccarello** [comment]
Thanks for the detailed report.

@nicknisi Since you implemented #449, do you have any idea why the lib option would look in the root directory?

**Comment by nicknisi** [comment]
I setup a quick test project on macOS with the same tsconfig.json and ran typedoc with the same args and got no errors. I setup the same thing on Windows and could reproduce, so it initially seems to be Windows-specific.
